libs/ui/inputs/src/lib/date-range-dropdown/date-range-dropdown.component.css

```diff
@@ -0,0 +1,5 @@
+:host {
+  .mat-date-range-input-container {
+    display: none !important;
```

Comment:
Is there another way to avoid the `!important`?

Reply:
Thanks for the remark, the `!important` is actually not needed anymore :-)